Sprint-1/exercises/random.js

```diff
@@ -7,3 +7,14 @@ const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 // Try breaking down the expression and using documentation to explain what it means
 // It will help to think about the order in which expressions are evaluated
 // Try logging the value of num and running the program several times to build an idea of what the program is doing
+
+/* 
+
+num represents a random number between 1 and 100.
+
+Math.random() returns a random floating-point number between 0 and 1.
```

Comment:
The phrase "between 0 and 1" alone is not precise enough in program specification because it does not state clearly whether 0 and 1 are included in the range.
 
Does the random value returned by `Math.random()` include 0? Does the return value include 1?

After you find out the answer, may I suggest asking ChatGPT how to precisely describe a range of numbers from 0 to 1 that include/exclude 0/1.

Reply:
Math.random() includes 0 but excludes 1.